shared/getting-started/nextSteps/_default.md

```diff
@@ -0,0 +1,12 @@
+- Try out [local mode][local-mode], the most efficient way to rapidly develop and test your {{ $names.company.lower }} application.
```

Comment:
I think `local mode` is a concept users might not be familiar with to begin with, so a more concrete explanation of it might be more appropriate.